http-server-netty/src/main/java/io/micronaut/http/server/netty/configuration/NettyHttpServerConfiguration.java

```diff
@@ -571,13 +733,6 @@ public void setPreferNativeTransport(boolean preferNativeTransport) {
             this.preferNativeTransport = preferNativeTransport;
         }
 
-        /**
-         * @return The number of threads to use
-         */
-        public int getNumOfThreads() {
-            return threads;
-        }
-
```

Comment:
Isn't this a breaking change? @graemerocher

Reply:
@jameskleeh didn't spot that, could you restore the method?